pkg/build/paths.go

```diff
@@ -21,9 +21,6 @@ func (ds DirNames) List() []string {
 type Paths struct {
 	// WorkDir is the absolute directory to run the build instructions in.
 	WorkDir string
-	// TargetDir is the absolute path to the dir where any other files
-	// needed to be included in the zip file should be placed.
-	TargetDir string
```

Comment:
`TargetDir` is now defined as a function that enforces the invariant that `TargetDir` is the dir containing `BinPath`.